### PR TITLE
Ssi audio sensor

### PR DESCRIPTION
**Pull request by murthyvedula**
Added audio data capture and recognition (Microphone sensor) to the SSI application project

### CODE REVIEW

File: qf_apps/qf_ssi_ai_app/sensor_audio/inc/sensor_audio_config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#define SENSOR_AUDIO_RATE_HZ_MAX     (16000)  ///< Maximum sample rate (in Hz) for this sensor
+#define SENSOR_AUDIO_MEMSIZE_MAX     (8192)
+#define SENSOR_AUDIO_CHANNELS        (1)
+
+/* Default settings for the sensor processing configurations.
+ * These may be overridden by user settings in the
+ * sensor_audio_config_user.h
+ */
+#ifndef SENSOR_AUDIO_RECOG_ENABLED
+#define SENSOR_AUDIO_RECOG_ENABLED 1    /* Enable SensiML recognition */
```
Comment (jublin) [inline qf_apps/qf_ssi_ai_app/sensor_audio/inc/sensor_audio_config.h:35]: Could we not utilize the existing macros for live vs recog?

Comment (murthyvedula) [reply qf_apps/qf_ssi_ai_app/sensor_audio/inc/sensor_audio_config.h:35]: Using the existing MACROs (SENSOR_SSSS_RECOG_ENABLED and SENSOR_SSSS_LIVESTREAM_ENABLED) would create dependency on sensor_ssss.h for the audio.

File: qf_apps/qf_ssi_ai_app/dcl_import.ssf [hunk 2]
```diff
@@ -65,6 +65,26 @@
                 }
             ]
         },
+        {
```
Comment (jublin) [inline qf_apps/qf_ssi_ai_app/dcl_import.ssf:68]: I think we duplicated microphones here.

Comment (murthyvedula) [reply qf_apps/qf_ssi_ai_app/dcl_import.ssf:68]: removed the duplicate entry

**Review by jublin — APPROVED** [review]

**Pull request merged**